File: Samples/Chess-board.py
from turtle import *

speed(100)
	
for i in range (8):
	up()
	goto (-320, -320+80*i)
	down()
	for j in range (8):
		if not (i + j) % 2:
			color ('black')
		else:
			color ('brown')
		begin_fill()
		for k in range (4):
			forward (80)
			left(90)
		forward (80)
		end_fill()

hideturtle()
mainloop()

File: test/Chess_Board.c
#include <turtle.h>

int main(void)
{	
    init();
    for (int i = 0; i < 2; i++)
    {
        up();
        goto(-160, -160 + 20 * i);
        down();
        for (int j = 0; j < 2; j++)
        {
            if (!((i + j) % 2))
                color("green");
            else
                color("white");

            begin_fill();
            for (int k = 0; k < 4; k++)
            {
                forward(20);
                left(90);
            }
            end_fill();
            forward(20);
        }
    }

    show();
    return 0;
}